cmake/VectorFlowConfig.cmake.in

```diff
@@ -0,0 +1,26 @@
+@PACKAGE_INIT@
+
+set_and_check(VectorFlow_INCLUDE_DIR "@PACKAGE_VectorFlow_INSTALL_INCLUDEDIR@")
+set(VecMath_FOUND True)
```

Comment:
Should we use [`find_dependency(VecMath)`](https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html?highlight=find_dependency#command:find_dependency) here?